homeassistant/components/frontend/www_static/micromarkdown/micromarkdown-js.html

```diff
@@ -0,0 +1,6 @@
+<script
+  type='text/javascript'
```

Comment:
Why would we refer to all these files from this file, resulting in not 1 but 4 requests to the server. Instead, have this file embed all the 3 files.